doc/source/conf.py

```diff
@@ -258,6 +258,13 @@ def check_python_bindings():
         [author_evenr],
         1,
     ),
+    (
+        "programs/gdal_raster_clean_collar",
+        "gdal-raster-collar",
```

Comment:
I suppose it is intentionally "gdal-raster-collar" and not "gdal-raster-clean-collar" in this place, but I still ask.

Reply:
good catch. will fix